docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -775,9 +775,89 @@ See below for a visual representation:
 
 **Birth Weight** 
 
-.. todo:: 
+Our model will apply the effect size of maternal consumption of iron fortified 
+foods on infant birth weight under the following assumptions:
+
+1. The effect of consuming iron fortified foods takes place six months after 
+the onset of consumption. This assumption is partially supported by Andersson 
+et al. 2010, which found that the peak response occurs at approximately six 
+months after onset; however, we simplified this finding for our model and 
+assumed that there was only a response six months after onset of consumpiton 
+of iron fortified foods.
+
+2. The effect of maternal iron fortified food consumption must occur at the 
+start of the third trimester of pregnancy in order to impact infant birth 
+weight, which we assumed was equal to 3 months prior to birth of the infant. 
+We chose to make this assumption because the effect of maternal iron was 
+measured at the third trimester in the Haider et al. 2013 paper.
+
+3. A simulant's birth weight will affect their all-cause mortality rate
+during the early and late neonatal periods only. This assumption is a 
+product of assumptions made in the modeling of the low birth weight and 
+short gestation (LBWSG) risk factor in GBD. 
+
+.. todo::
+
+	Cite the LBWSG risk factor page once created.
+
+Therefore, for our simulation, an infant's mother must have gained coverage 
+to iron fortified foods at least **nine months prior to the birth of the 
+infant** in order for the iron fortification coverage to affect the infant's 
+birth weight.
+
```

Comment:
oh wait...how is this different to assumption 2? 
so for an infants birthweight to be increased, coverage must have occurred 9 months prior, agreed. 
then what is assumption 2 saying? is it that woman needs to be eating fortified food at start of pregnancy, takes 6 months to be effective, so the effect kicks in at start of 3rd trimester?

Reply:
Yeah, you're understanding that correctly. See my comments in the thread above :)